docs/common/craft-parts/reference/plugins/poetry_plugin.rst

```diff
@@ -0,0 +1,106 @@
+.. _craft_parts_poetry_plugin:
+
+Poetry plugin
+=============
+
+**Version**: 2.1
```

Comment:
I wonder if this is relevant? Since this document is part of craft-parts, if a given application has this reference page then its version of craft-parts must have the poetry plugin right?

Might sound pedantic but I'm thinking about the point of view of a, say, Snapcraft user; there isn't really anything that they can do with this `2.1` since they typically don't know which version of craft-parts their Snapcraft is using

Reply:
Agreed and removed. I had it there because of [the plugin documentation howto](https://canonical-craft-parts.readthedocs-hosted.com/en/latest/how-to/document-plugin.html). Do you think we should remove it from there?

Reply:
yeap

Reply:
https://github.com/canonical/craft-parts/pull/831